doc/guide/guide-control.rst

```diff
@@ -191,65 +191,7 @@ algorithm.
-  There is a specific subclass that is used by the CRAB algorithm to generate the pulses based on the basis coefficients that are being optimised.
-
-TerminationConditions
-  This is simply a convenient place to hold all the properties that will determine when the single optimisation run terminates. Limits can be set for number of iterations, time, and of course the target infidelity.
-
-Stats
-  Performance data are optionally collected during the optimisation. This object is shared to a single location to store, calculate and report run statistics.
-
-FidelityComputer
-  The subclass of the fidelity computer determines the type of fidelity measure. These are closely linked to the type of dynamics in use. These are also the most commonly user customised subclasses.
-
-PropagatorComputer
-  This object computes propagators from one timeslot to the next and also the propagator gradient. The options are using the spectral decomposition or Frechet derivative, as discussed above.
-
-TimeslotComputer
-  Here the time evolution is computed by calling the methods of the other computer objects.
-
-OptimResult
-  The result of a pulse optimisation run is returned as an object with properties for the outcome in terms of the infidelity, reason for termination, performance statistics, final evolution, and more.
-
-.. _pulseoptim-functions:
-
-Using the pulseoptim functions
-==============================
-The simplest method for optimising a control pulse is to call one of the functions in the ``pulseoptim`` module. This automates the creation and configuration of the necessary objects, generation of initial pulses, running the optimisation and returning the result. There are functions specifically for unitary dynamics, and also specifically for the CRAB algorithm (GRAPE is the default). The ``optimise_pulse`` function can in fact be used for unitary dynamics and / or the CRAB algorithm, the more specific functions simply have parameter names that are more familiar in that application.
-
-A semi-automated method is to use the ``create_optimizer_objects`` function to generate and configure all the objects, then manually set the initial pulse and call the optimisation. This would be more efficient when repeating runs with different starting conditions.
+The Quantum Control part of qutip has been moved to it's own project.
+Previously available implementation is now located in the `qutip-qtrl <https://qutip-qtrl.readthedocs.io/en/stable/>`_ module.
+A newer interface with upgraded capacities is also being developped in `qutip-qoc <https://github.com/flowerthrower/qutip-qoc/tree/packaging>`_.
+Please give these module a look.
```

Comment:
```suggestion
The Quantum Control part of qutip has been moved to its own project.

The previously available implementation is now located in the `qutip-qtrl <https://qutip-qtrl.readthedocs.io/en/stable/>`_ module. If the ``qutip-qtrl`` package is installed, it can also be imported under the name ``qutip.control`` to ease porting code developed for QuTiP 4 to QuTiP 5.

A newer interface with upgraded capacities is being developped in `qutip-qoc <https://github.com/flowerthrower/qutip-qoc/tree/packaging>`_.

Please give these modules a try.
```